src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/ResourceLoader/AccountResourceLoader.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Infrastructure\Sulu\Content\ResourceLoader;
+
+use Sulu\Bundle\ContactBundle\Api\Account as AccountApi;
+use Sulu\Bundle\ContactBundle\Contact\ContactManagerInterface;
+use Sulu\Bundle\ContactBundle\Entity\AccountAddress as AccountAddressEntity;
+use Sulu\Bundle\ContactBundle\Entity\AccountInterface;
+use Sulu\Bundle\ContentBundle\Content\Application\ResourceLoader\ResourceLoaderInterface;
+
+class AccountResourceLoader implements ResourceLoaderInterface
+{
+    public const RESOURCE_LOADER_KEY = 'account';
+
+    /**
+     * @param ContactManagerInterface<AccountInterface, AccountApi, AccountAddressEntity> $accountManager
+     */
+    public function __construct(
+        private ContactManagerInterface $accountManager,
```

Comment:
```suggestion
        private AccountManagerInterface $accountManager,
```

Reply:
There is no accountmanagerinterface contact and account manager use the same interface

Reply:
🙈